packages/gatsby-theme-carbon/src/styles/internal/_carbon-components.scss

```diff
@@ -1,43 +1,43 @@
-@import '~carbon-components/scss/components/skeleton/skeleton';
-@import '~carbon-components/scss/components/inline-loading/inline-loading';
-@import '~carbon-components/scss/components/pagination-nav/pagination-nav';
-@import '~carbon-components/scss/components/ui-shell/ui-shell';
+// @use '@carbon/react/scss/components/button/button';
+// @use '@carbon/react/scss/components/copy-button/copy-button';
+// @use '@carbon/react/scss/components/file-uploader/file-uploader';
+// @use '@carbon/react/scss/components/checkbox/checkbox';
+// @use '@carbon/react/scss/components/combo-box/combo-box';
+// @use '@carbon/react/scss/components/radio-button/radio-button';
+// @use '@carbon/react/scss/components/toggle/toggle';
+// @use '@carbon/react/scss/components/search/search';
+// @use '@carbon/react/scss/components/select/select';
+// @use '@carbon/react/scss/components/text-input/text-input';
+// @use '@carbon/react/scss/components/text-area/text-area';
+// @use '@carbon/react/scss/components/number-input/number-input';
+// @use '@carbon/react/scss/components/form/form';
+// @use '@carbon/react/scss/components/link/link';
+// @use '@carbon/react/scss/components/list-box/list-box';
+// @use '@carbon/react/scss/components/list/list';
+// @use '@carbon/react/scss/components/data-table/data-table';
+// @use '@carbon/react/scss/components/structured-list/structured-list';
+// @use '@carbon/react/scss/components/code-snippet/code-snippet';
+// @use '@carbon/react/scss/components/overflow-menu/overflow-menu';
+// @use '@carbon/react/scss/components/content-switcher/content-switcher';
+// @use '@carbon/react/scss/components/date-picker/date-picker';
+// @use '@carbon/react/scss/components/dropdown/dropdown';
+// @use '@carbon/react/scss/components/loading/loading';
+// @use '@carbon/react/scss/components/modal/modal';
+// @use '@carbon/react/scss/components/multi-select/multi-select';
+// @use '@carbon/react/scss/components/notification/inline-notification';
+// @use '@carbon/react/scss/components/notification/toast-notification';
+// @use '@carbon/react/scss/components/tooltip/tooltip';
+// @use '@carbon/react/scss/components/tabs/tabs';
+// @use '@carbon/react/scss/components/tag/tag';
+// @use '@carbon/react/scss/components/pagination/pagination';
+// @use '@carbon/react/scss/components/accordion/accordion';
+// @use '@carbon/react/scss/components/progress-indicator/progress-indicator';
+// @use '@carbon/react/scss/components/breadcrumb/breadcrumb';
+// @use '@carbon/react/scss/components/toolbar/toolbar';
+// @use '@carbon/react/scss/components/time-picker/time-picker';
+// @use '@carbon/react/scss/components/slider/slider';
+// @use '@carbon/react/scss/components/tile/tile';
+// @use '@carbon/react/scss/components/skeleton/skeleton';
+// @use '@carbon/react/scss/components/inline-loading/inline-loading';
+// @use '@carbon/react/scss/components/pagination-nav/pagination-nav';
+// @use '@carbon/react/scss/components/ui-shell/ui-shell';
```

Comment:
Did we need to have these in, or are the per-component styles brought in okay through the main entrypoint?